etcd/vendor/github.com/openshift/microshift/pkg/config/multinode.go

```diff
@@ -0,0 +1,22 @@
+package config
+
+type MultiNode struct {
+	Enabled bool `json:"enabled"`
+	// only one controlplane node is supported
+	// IP address of control plane node
+	Controlplane string `json:"controlplane"`
+}
+
+// MultinodeConfig populates multinode configurations to Config.MultiNode
+func (c *Config) MultinodeConfig(enabled bool) {
```

Comment:
```suggestion
func (c *Config) MultiNode(enabled bool) {
```

It might be good to have this function return a new Config instance, so it's clear that any transformation is one way. We can do that in another PR, though.